hybris/platforms/common/server_wlegl.h

```diff
@@ -43,4 +43,7 @@ server_wlegl_create(struct wl_display *wldpy);
 void
 server_wlegl_destroy(server_wlegl *wlegl);
 
+server_wlegl *
+server_wlegl_from(struct wl_resource *);
```

Comment:
Was there some reason to add this to header? It is only used in server_wlegl.cpp.

Reply:
that was simplifying testing to me, nothing more. left it here just for consistency with other headers.
should I move it to .cpp?